client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -188,9 +188,32 @@ const BrewRenderer = (props)=>{
 		document.dispatchEvent(new MouseEvent('click'));
 	};
 
-	const rendererPath  = props.renderer == 'V3' ? 'V3' : 'Legacy';
-	const themePath     = props.theme ?? '5ePHB';
-	const baseThemePath = Themes[rendererPath][themePath].baseTheme;
+	let rendererPath  = props.renderer == 'V3' ? 'V3' : 'Legacy';
+	let baseRendererPath = props.renderer == 'V3' ? 'V3' : 'Legacy';
+	const blankRendererPath = props.renderer == 'V3' ? 'V3' : 'Legacy';
```

Comment:
Seems like all 3 of these end up being the same value. Can we just use one variable for this? Themes should be restricted to working with a matching renderer engine anyway. I.e., a V3 brew cannot use a Legacy Theme.

Reply:
I don't recall doing that for a specific reason at the time. I may have kept them distinct in case they proved to need be in the future.

Reviewing the code, there was some redundancy and one of the consts only existed to be more readable. Collapsed that.